### DISCUSSION TITLE
Additional Info for Help-file

### DISCUSSION THREAD

**Discussion by wsc44** [GrandOrgue/OdfEdit#53] [Ideas]
I think it would be a good idea to amend the help file with additional information concerning the conversion parameters. Think of a newbie, not well versed in odf-files and not reading the OdfEdit and/or GO-threads on github. So, for the various input parameter options the following questions come to my mind: 
Save ODF in ISO-8859-1 format (else UTF-8-com): Why should i choose that option? Is there a program/application that would need the ISO-format?
HW to GO - do not convert key noises: Is there a special reason why someone wouldn't/shouldn't want to convert the noises, considering that they usually can be adjusted/shut off via the standard GO menu entry "Coupler manuals and volume"?

HW to GO - place tremmed samples in separate ranks: What is the advantage/disadvantage of placing the tremmed samples in separate ranks? 

HW to GO - correct pipes pitch from metadata or file name: I gather that the correct conversion parameter depends on the specific characteristics of the HW-Odf and that OdfEdit cannot decide for itself which way to go. Is that correct? If so maybe state that info in a header text on top of both options. - What would be the best method for a user to decide which of both options to choose? First try out "metadata", if that doesn't produce conversion messages in log file than try option "file name"?  

Hope these questions give a rough idea of the kind of information that would be helpful for someone new. Thanks.

**Comment by eturpault** [comment]
You are right the help about the options could be more elaborated for newbies.
About the ISO-8859-1 format, I don't know the interrest in using it, but as GO supports it and @oleg68 requested this option in [this discussion](https://github.com/GrandOrgue/OdfEdit/discussions/18#discussion-5066925) I added it. UTF-8 is definitively better to support special characters in the texts of the ODF and is the default format used by OdfEdit.
@oleg68 : which interest has the ISO-8859-1 format according to you ?

About the keys noises, personnaly I don't like to hear them as the keyboard of my organ has already its own keys noises. So this option permits to have by default none keys noises, without having to shut them off in the GO menu. Several sample sets have stop noises and keys noises in the same WindchestGroup, if one wants to have the first and not the second, this options is good for that.

For others options, I will think to what I can add to the existing help.

**Comment by eturpault** [comment]
The help of OdfEdit v2.12 has been enriched as requested in this discussion.
